dashboard/app/helpers/levels_helper.rb

```diff
@@ -771,7 +771,7 @@ def lab2_options
     app_options[:public_caching] = @public_caching
     if @script_level&.lesson
       app_options[:theme] = @script_level.lesson.get_background_for_user(current_user)
-    elsif @level.is_a?(Pythonlab) && current_user
+    elsif @level.uses_theme_preference? && current_user
```

Comment:
Just curious - with this change, the user sets a global theme for labs so that same user theme preference is set for both python lab and weblab2. As we allow more labs to be theme-able, do we envision maybe allowing setting a theme preference per lab?

Reply:
Our goal is to have one theme preference across the whole site (eventually).